apps/docs/app/(home)/page.tsx

```
@@ -195,3 +193,34 @@ const Logos = () => {
+
+  return (
+    <button
+      onClick={copyToClipboard}
+      className={buttonVariants({
+        size: "lg",
+        variant: "outline",
+        className:
+          "bg-background font-bold group relative flex h-12 items-center gap-2 rounded-lg border px-4 py-3 font-mono text-sm transition-all",
+      })}
+    >
+      <span>$ npx assistant-ui init</span>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)